src/data.compiled.json

```diff
@@ -936,6 +933,14 @@
       "min": 0,
       "max": 1000
     },
+    "tx_queue_locals": {
+      "name": "Local Accounts",
+      "type": "hashset<string>",
```

Comment:
Is there some way to get the script to recognize this as a `string[]` like in the other similar config params?

Reply:
Parity Ethereum only accepts `tx_queue_locals = "account1,account2"` in the config, and not `tx_queue_locals = "["account1","account2"]"` though, right?

Reply:
I guessed based on this: 

https://github.com/paritytech/parity-ethereum/blob/4cc274e75f2c4e86bbe7fd42bcee816f08c926de/parity/cli/tests/config.full.toml#L133

it should be `["account1","account2"]` unless the format is `["account1,account2"]`

Reply:
oki, then I think it should be safe to add `"type": "string[]"` for `tx_queue_locals` in `data.extra.json`

or did you mean you wanted to find a way to systematically interpret `hashset<...>` as arrays?

Reply:
I'll do that, thanks, wasn't sure if it was the right way to add the property because we don't use it anywhere else. 

> or did you mean you wanted to find a way to systematically interpret hashset<...> as arrays?

I'm not sure - I don't imagine any other config params will use this in the future.